recipes/psy-maps/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "psy-maps" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "f2b1682467bd7af2b25f856935f2af8476ada8831f0935635b6e0816d890aa08" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+  - python
+  - setuptools
+  run:
+  - python
+  - setuptools
```

Comment:
Do you need `setuptools` at run time? (Sometimes we do need when packages use `pkg_resources`. If not we can remove it.)